nautobot/extras/choices.py

```diff
@@ -19,16 +19,19 @@ class ApprovalWorkflowStateChoices(ChoiceSet):
     PENDING = "Pending"
     APPROVED = "Approved"
     DENIED = "Denied"
+    CANCELED = "Canceled"
 
     CHOICES = (
         (PENDING, "Pending"),
         (APPROVED, "Approved"),
         (DENIED, "Denied"),
+        (CANCELED, "Canceled"),
     )
     CSS_CLASSES = {
         PENDING: "info",
         APPROVED: "success",
         DENIED: "danger",
+        CANCELED: "canceled",
```

Comment:
This isn't a current CSS class is it? Perhaps:
```suggestion
        CANCELED: "warning",
```